sdk/core/azure-core-amqp/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.8.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <version>1.8.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;current} -->
```

Comment:
Since Event Hubs depends on implementation classes in azure-core-amqp. This will break it. Have you checked to make sure it works there?